docs/reference/ingest/enrich.asciidoc

```diff
@@ -249,9 +249,16 @@ enrich policy executor.
 The enrich coordinator supports the following node settings:
 
 `enrich.cache_size`::
-Maximum number of searches to cache for enriching documents. Defaults to `1000`.
-There is a single cache for all enrich processors in the cluster. This setting
-determines the size of that cache.
+Maximum size of the cache that caches searches for enriching documents.
+The size can be specified in three units: the raw number of
+cached searches (e.g. `1000`), an absolute size in bytes (e.g. `100Mb`),
```

Comment:
Should we explicitly state here that you need to in include a `b` (upper or lowercase) for the second unit?

Reply:
I think we should allow all the same things that `ByteSizeValue` supports, which would be all the endings in https://github.com/elastic/elasticsearch/blob/ffc22b2a8a503e387cc029df632a97e93f453566/server/src/main/java/org/elasticsearch/common/unit/ByteSizeValue.java#L232-L252

Reply:
I've updated the logic to handle all of `ByteSizeValue`'s cases. But I was more referring to that if you want to specify bytes, you'll have to add one of those letters -- since specifying a raw number of bytes would be parsed as a flat document count. But that disclaimer might be redundant/overkill.